apps/worker/Dockerfile

```
@@ -17,31 +17,47 @@ COPY packages/queue/package.json ./packages/queue/
 RUN bun build \
     --minify-whitespace \
     --minify-syntax \
     --target bun \
     --outfile ./dist/index.js \
     ./apps/worker/src/index.ts
 
-FROM oven/bun:alpine AS runner
+FROM docker.io/oven/bun:alpine AS runner
 WORKDIR /app
 
+RUN apk add --no-cache procps
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)